proto/v1/project_service.proto

```diff
@@ -192,6 +190,9 @@ message ListProjectsResponse {
 message SearchProjectsRequest {
   // Show deleted projects if specified.
   bool show_deleted = 1;
+
+  // The project title or resource id.
+  string search = 2;
```

Comment:
maybe "query" is better?

https://cloud.google.com/resource-manager/reference/rest/v3/projects/search

Reply:
LGTM